cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/leveldb/LevelDBStreamAdmin.java

```diff
@@ -46,9 +46,8 @@ public LevelDBStreamAdmin(CConfiguration conf, LevelDBOrderedTableService servic
   @Override
   public String getActualTableName(QueueName queueName) {
     if (queueName.isStream()) {
-      //TODO: namespace the tableName (have to modify tableNamePrefix)
-      // <cdap namespace>.system.stream.<stream name>
-      return getTableNamePrefix() + "." + queueName.getSecondComponent();
+      // <cdap namespace>.system.stream.<namespace>.<stream name>
+      return getTableNamePrefix() + "." + queueName.getFirstComponent() + "." + queueName.getSecondComponent();
```

Comment:
hmmm, worth pulling this up into an AbstractStreamAdmin? Again, should do it for queues as well.
Also, need the hack here for not adding the first component for default namespaces.